### DISCUSSION TITLE
know more about code snippets idea

### DISCUSSION THREAD

**Discussion by MOHAMED-EHAB-DEV** [reactplay/react-play#173] [Q&A] [answered]
@atapas

**Comment by atapas** [comment, marked as answer]
It was initially cited by @murtuzaalisurti as an idea so that he could add more. 

What I was thinking in parallel was... Code snippets of most used or frequently asked questions from popular programming languages like Java, Python, JavaScript. The snippet can be downloaded as image and can be copied as text as well.


@murtuzaalisurti please add your thoughts as well.

**Reply by murtuzaalisurti** [reply]
Nice idea @atapas! We can do something like 30 seconds of code and provide a list of general snippets so that a developer does not have to re-invent the wheel while at the same time give an option to users to add their own snippets in order to download and share them with other developers!

**Reply by MOHAMED-EHAB-DEV** [reply]
But I want to some developers to work with me because i can't make it by myself + I will share it as a link not in reactplay

**Reply by atapas** [reply]
> But I want to some developers to work with me because i can't make it by myself + I will share it as a link not in reactplay

ReactPlay doesn't support providing as a link. We want to build it as part of the platform.

**Reply by MOHAMED-EHAB-DEV** [reply]
> > But I want to some developers to work with me because i can't make it by myself + I will share it as a link not in reactplay
> 
> ReactPlay doesn't support providing as a link. We want to build it as part of the platform.

I will try

**Comment by MOHAMED-EHAB-DEV** [comment]
Thanks @atapas and @murtuzaalisurti 

I understand now

**Reply by atapas** [reply]
Alright. Marking as answer give,
